### PR TITLE
Add iOS styles

### PR DESCRIPTION
**Pull request by rub**
**What it does**

Adds iOS icons and general styles.

### CODE REVIEW

File: Lumino/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { StyleSheet } from 'react-native';
-import SplashScreen from './js/SplashScreen';
+import { StyleSheet, View } from 'react-native';
```
Comment (silvanocerza) [inline Lumino/App.js]: `View` is unused.

File: Lumino/js/ControllersList.js [hunk 2]
```diff
@@ -1,17 +1,31 @@
 import React from 'react';
-import { Button, StyleSheet, ScrollView } from 'react-native';
+import { StyleSheet, ScrollView, Text, TouchableOpacity } from 'react-native';
```
Comment (silvanocerza) [inline Lumino/js/ControllersList.js]: `Text` is not used.

File: Lumino/js/NewGatewayForm.js [hunk 3]
```diff
@@ -17,35 +24,27 @@ export default class NewGatewayForm extends React.Component {
...
+        />
+        <TextInput
+          style={styles.textInput}
+          placeholder="Port"
+          onChangeText={(text) => this.setState({text})}
+        />
+        <TextInput
+          style={styles.textInput}
+          placeholder="Password"
+          secureTextEntry
```
Comment (silvanocerza) [inline Lumino/js/NewGatewayForm.js]: Would be great if you are explicit here when settings `secureTextEntry`.

**Review by silvanocerza — CHANGES_REQUESTED** [review]
Check comments

File: Lumino/js/NewGatewayForm.js (hunk 3 above)
Comment (rub) [reply Lumino/js/NewGatewayForm.js]: If you need to set an attribute value to `true` in JSX, you can omit the value.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md

**Review by silvanocerza — APPROVED** [review]

**Pull request merged**